### PR TITLE
Networking HTTP/2 RN update

### PR DESCRIPTION
**Pull request by bmcelvee**
Release note update. Will link the procedure module in separate PR. 

@codyhoag FYI

### CODE REVIEW

**Comment by bmcelvee** [comment]
Procedure PR: https://github.com/openshift/openshift-docs/pull/23571

**Comment by bmcelvee** [comment]
@Miciah @frobware @mcurry-rh @knobunc here's the PR for the revised release note. 

I'm pulling the procedure out of the release notes and into it's own module: https://github.com/openshift/openshift-docs/pull/23571

**Comment by bmcelvee** [comment]
The link might cause the build to fail because the PR has not yet merged.

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -1819,6 +1825,11 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+|HTTP/2 Support
+|-
+|-
+|GA
+
```
Comment (Miciah) [inline release_notes/ocp-4-5-release-notes.adoc:1832]: I don't think we need this entry in the table if the feature was never labeled "TP".

**Comment by jboxman** [comment]
@bmcelvee, LGTM. Don't forget to wire-up the `xref` once that page is live.

**Pull request merged**